taskt/Core/Automation/Commands/Web Browser Commands/SeleniumElementActionCommand.cs

```diff
@@ -787,64 +712,61 @@ public void seleniumAction_SelectionChangeCommitted(object sender, EventArgs e)
 
                     //assign cell as a combobox
                     if (sender != null)
-                    {
-                        ElementsGridViewHelper.Rows[0].Cells[1].Value = "Select By Text";
-                    }
-
-                    ElementsGridViewHelper.Rows[0].Cells[1] = selectionTypeBox;
-
+                        _elementsGridViewHelper.Rows[0].Cells[1].Value = "Select By Text";
 
+                    _elementsGridViewHelper.Rows[0].Cells[1] = selectionTypeBox;
                     break;
+
                 case "Wait For Element To Exist":
-                    foreach (var ctrl in ElementParameterControls)
-                    {
+                    foreach (var ctrl in _elementParameterControls)
                         ctrl.Show();
-                    }
+       
                     if (sender != null)
-                    {
                         actionParameters.Rows.Add("Timeout (Seconds)");
-                    }
                     break;
 
                 case "Switch to frame":
-                    foreach (var ctrl in ElementParameterControls)
-                    {
+                    foreach (var ctrl in _elementParameterControls)
                         ctrl.Hide();
-                    }
                     break;
 
                 default:
                     break;
             }
-
-            ElementsGridViewHelper.DataSource = v_WebActionParameterTable;
+            _elementsGridViewHelper.DataSource = v_WebActionParameterTable;
         }
 
-        private void ElementsGridViewHelper_CellContentClick(object sender, DataGridViewCellEventArgs e)
+        private void _elementsGridViewHelper_CellContentClick(object sender, DataGridViewCellEventArgs e)
```

Comment:
PascalCase naming convention for Event Handlers.

Reply:
done